tests/acceptance/17_packages/10_new/unsafe/packagesmatching.cf.sub

```diff
@@ -0,0 +1,23 @@
+# Test that updates list cache is not updated too often.
+
+body common control
+{
+    inputs => { "../../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+
+    package_module => "test_module";
```

Comment:
I actually don't see anything "unsafe" in this test, you are not modifying the real packages or anything. So I think you can move this out of the `unsafe` directory, instead into the `timed` directory.